<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connect to server</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_port">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>10101</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBox_time">
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>107</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Search time(seconds):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
